spi-extension/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.misarch.keycloak.event.provider</groupId>
+    <artifactId>keycloak-dapr-events</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+
+    <name>Keycloak: Event Publisher to dapr</name>
+    <properties>
+        <keycloak.version>23.0.6</keycloak.version>
```

Comment:
I can foresee problems when updating keycloak as devs might not know they need to update both `Dockerfile` and this `pom.xml`.
Perhaps we should document that in the `README`?

Reply:
added with a3fa0964a328114419bdb9c13e5f5bed0f4a6b81